ipa-core/src/helpers/mod.rs

```diff
@@ -555,6 +555,18 @@ impl From<usize> for TotalRecords {
     }
 }
 
+pub fn zip3<T, U>(a: [T; 3], b: [U; 3]) -> [(T, U); 3] {
```

Comment:
we may want to have a module higher up the tree to put these in

Reply:
Do you have a suggestion? There are a bunch of other utilities stashed within helpers that also aren't exclusive to the infrastructure. If not for the disruption of renaming things, I might rename `helpers` to `infra` and then create something like `utils` for the utilities.

Reply:
yea I agree that these will be useful outside of infra. Renaming `helpers` to `infra` is long overdue and as much as I don't like `utils` name, I don't have a better suggestion for this module so I think `utils` will be good with internal modules like `array`, `collection` , `stream` etc

Reply:
Sounds good. I'm not going to move anything beyond the two new functions right now, but we can move more later.